apps/researcher/src/app/[locale]/objects/[id]/date-created-to-string.ts

```diff
@@ -0,0 +1,24 @@
+import {TimeSpan} from '@/lib/api/definitions';
+
+export default function dateCreatedToString(
```

Comment:
Would it be an option to not use `default`? That way we can add new functions to the file.

Reply:
I prefer `export default` for single-function files. But this could be because this rule was part of the Airbnb eslint config I'm more used to working with. Choosing `export default` for single-function files is a more straightforward rule. 

I have no problem changing this, but then, for me, I'm not sure when you prefer `export default` and when `export`. Can you explain your preference some more? For example, do you prefer' export' for single component files? I was not planning to add extra functions to this file. So for me, `export default` makes more sense.

Reply:
I prefer to not use `export default` because of its side-effects (see e.g. https://www.lloydatkinson.net/posts/2022/default-exports-in-javascript-modules-are-terrible/), unless a framework wants it so (e.g. React components).